clients/geth/specular/rollup/client/l1client.go

```diff
@@ -32,7 +32,7 @@ type L1BridgeClient interface {
 	) event.Subscription
 	Close()
 	// ISequencerInbox.sol
-	AppendTxBatch(contexts []*big.Int, txLengths []*big.Int, txBatch []byte) (*types.Transaction, error)
+	AppendTxBatch(contexts []*big.Int, txLengths []*big.Int, firstL2Block *big.Int, txBatch []byte) (*types.Transaction, error)
```

Comment:
nit for clarity (here and other places)
```suggestion
	AppendTxBatch(contexts []*big.Int, txLengths []*big.Int, firstL2BlockNumber *big.Int, txBatch []byte) (*types.Transaction, error)
```